logtail/logtail.go

```diff
@@ -757,6 +767,33 @@ func (l *Logger) Write(buf []byte) (int, error) {
 	return inLen, err
 }
 
+func redactIPs(buf []byte) []byte {
```

Comment:
Although it's unexported, this function is doing enough interesting work that it probably deserves a doc comment. In particular it'd be good to say what the input is and to summarize how the function transforms it.